.github/workflows/cd.yml

```
@@ -63,8 +63,38 @@ jobs:
           fail_ci_if_error: false
           token: ${{ secrets.CODECOV_TOKEN }} # not required for public repos
 
+  core-test-win:
+    needs: prep-testbed
+    runs-on: windows-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.7]
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python ${{ matrix.python-version}}
+        uses: actions/setup-python@v3
+        with:
+          python-version: ${{ matrix.python-versionn}}
+      - name: Prepare environment
+        shell: pwsh
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install wheel
+          pip install --pre docarray
+          pip install -e ".[test]"
+      - name: Test
+        id: test
+        shell: pwsh
+        run: |
+          $env:PYTHONIOENCODING='utf-8'
+          cd tests/
+          pytest --suppress-no-test-exit-code --cov=annlite --cov-report=xml -v -s -m "not gpu" -k "test"
+          echo "::set-output name=codecov_flag::annlite"
```

Please turn off the coverage report in the pytest on windows. And what's more, the target test cases should come from `${{ matrix.test-path }}`

```suggestion
          pytest -v -s -m "not gpu" ${{ matrix.test-path }}
```

---

`${{ matrix.test-path }}` contains `*`, however asterisk wildcard `*` is not supported in pytest on windows.